Class3.cs

```diff
@@ -0,0 +1,97 @@
+        ConsoleColor originalColor = Console.ForegroundColor;
+
+        Console.WriteLine("=== Стихотворение с разноцветным выводом ===\n");
+
+        // Вывод автора - зеленым цветом
+        Console.ForegroundColor = ConsoleColor.Green;
+        Console.WriteLine("Автор: Александр Пушкин");
+
+        // Вывод названия - желтым цветом
+        Console.ForegroundColor = ConsoleColor.Yellow;
+        Console.WriteLine("Название: \"Я помню чудное мгновенье...\"");
+        Console.WriteLine();
+
+        // Вывод строфы стихотворения разными цветами
+        Console.ForegroundColor = ConsoleColor.Cyan;
+        Console.Write("Я помню ");
+
+        Console.ForegroundColor = ConsoleColor.Magenta;
+        Console.Write("чудное мгновенье: ");
+
+        Console.ForegroundColor = ConsoleColor.White;
+        Console.WriteLine();
+
+        Console.ForegroundColor = ConsoleColor.Blue;
+        Console.Write("Передо мной ");
+
+        Console.ForegroundColor = ConsoleColor.Red;
+        Console.Write("явилась ты, ");
+
+        Console.ForegroundColor = ConsoleColor.White;
+        Console.WriteLine();
+
+        Console.ForegroundColor = ConsoleColor.Cyan;
+        Console.Write("Как мимолетное ");
+
+        Console.ForegroundColor = ConsoleColor.Yellow;
+        Console.Write("виденье, ");
+
+        Console.ForegroundColor = ConsoleColor.White;
+        Console.WriteLine();
+
+        Console.ForegroundColor = ConsoleColor.Green;
+        Console.Write("Как гений ");
+
+        Console.ForegroundColor = ConsoleColor.Magenta;
+        Console.Write("чистой красоты.");
+
+        Console.ForegroundColor = ConsoleColor.White;
+        Console.WriteLine("\n");
+
+        // Вторая строфа
+        Console.ForegroundColor = ConsoleColor.DarkCyan;
+        Console.Write("В томленьях ");
+
+        Console.ForegroundColor = ConsoleColor.DarkYellow;
+        Console.Write("грусти безнадежной, ");
+
+        Console.ForegroundColor = ConsoleColor.White;
+        Console.WriteLine();
+
+        Console.ForegroundColor = ConsoleColor.DarkGreen;
+        Console.Write("В тревогах ");
+
+        Console.ForegroundColor = ConsoleColor.DarkRed;
+        Console.Write("шумной суеты, ");
+
+        Console.ForegroundColor = ConsoleColor.White;
+        Console.WriteLine();
+
+        Console.ForegroundColor = ConsoleColor.DarkBlue;
+        Console.Write("Звучал мне ");
+
+        Console.ForegroundColor = ConsoleColor.DarkMagenta;
+        Console.Write("долго голос нежный ");
+
+        Console.ForegroundColor = ConsoleColor.White;
+        Console.WriteLine();
+
+        Console.ForegroundColor = ConsoleColor.Gray;
+        Console.Write("И снились ");
+
+        Console.ForegroundColor = ConsoleColor.Cyan;
+        Console.Write("милые черты.");
+
+        // Восстанавливаем оригинальный цвет консоли
+        Console.ForegroundColor = originalColor;
+        Console.WriteLine("\n\nНажмите любую клавишу для выхода...");
+        Console.ReadKey();
+    }
```

Comment:
В конце программы хорошо бы восстановить цвета консоли
```suggestion
        Console.ResetColor();
        Console.ReadKey();
    }
```